Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs

```diff
@@ -462,6 +466,28 @@ public IHttpActionResult GetEditionComparitor()
             foreach (Edition edition in Enum.GetValues(typeof(Edition))) editionComparitor.Add(edition.ToString(), EditionChecker.CheckEdition(edition));
             return Ok(editionComparitor);
         }
+
+        //Note: if we make a SCADA point model controller, we may want move this to that
+        [Route("SCADAPoint/SCADAPointSearch"), HttpPost]
+        public async Task<IHttpActionResult> QuerySCADADataPoints([FromBody] JObject query, CancellationToken token)
+        {
+            try
+            {
+                SystemCenterXDAAPIHelper.RefreshSettings();
+                using (HttpResponseMessage response = helper
+                    .GetResponseTask($"api/SystemCenter/SCADAPoint/SCADAPointSearch", new StringContent(query.ToString(), Encoding.UTF8, "application/json"))
+                    .Result)
+                {
+                    response.EnsureSuccessStatusCode();
+                    string result = await response.Content.ReadAsStringAsync();
+                    return Ok(result);
+                };
+            }
+            catch (Exception ex)
+            {
+                return InternalServerError(ex);
+            }
```

Comment:
I think this get's handled by the base class
```suggestion
                SystemCenterXDAAPIHelper.RefreshSettings();
                using (HttpResponseMessage response = helper
                    .GetResponseTask($"api/SystemCenter/SCADAPoint/SCADAPointSearch", new StringContent(query.ToString(), Encoding.UTF8, "application/json"))
                    .Result)
                {
                    response.EnsureSuccessStatusCode();
                    string result = await response.Content.ReadAsStringAsync();
                    return Ok(result);
                };           
```